xbmc/cores/dvdplayer/DVDInputStreams/DVDFactoryInputStream.cpp

```diff
@@ -45,6 +45,8 @@ CDVDInputStream* CDVDFactoryInputStream::CreateInputStream(IDVDPlayer* pPlayer,
 {
   CFileItem item(file.c_str(), false);
 
+  item.SetMimeType(content);
```

Comment:
It's not needed until later but it felt reasonable to set when we construct our fileitem